eng/pipelines/templates/stages/archetype-sdk-tests.yml

```diff
@@ -99,6 +99,8 @@ stages:
         jobs:
         - template: /eng/common/pipelines/templates/jobs/archetype-sdk-tests-generate.yml
           parameters:
+            SparseCheckoutPaths:
+              - "sdk/${{ parameters.ServiceDirectory }}/**/*.json"
```

Comment:
Might be worth a comment explaining that this implicitly excludes the yaml based recording files?